explainaboard/builders/kg_link_tail_prediction.py

```diff
@@ -0,0 +1,241 @@
+from typing import Optional
+import os
+import sys
+import spacy
+spacy_nlp = spacy.load("en_core_web_sm")
```

Comment:
We just refactored the way to use spacy model a little bit, it would be nice if you can also modify based on this example:
https://github.com/neulab/ExplainaBoard/blob/1febb198fb01b2d615611dcf42717988786942d4/explainaboard/builders/text_classification.py#L10

Reply:
Refactored as of the latest commit